src/test/resources/org/jenkinsci/plugins/matrixauth/integrations/configuration-as-code.yml

```diff
@@ -0,0 +1,73 @@
+jenkins:
+  authorizationStrategy:
+    projectMatrix:
```

Comment:
may have missed it but is there a test needed for globalMatrix?

Reply:
Would consider that redundant due to how inheritance is set up.